addons/website_sale/models/website.py

```diff
@@ -79,6 +79,26 @@ def _get_product_sort_mapping():
         ],
         default='optional')
 
+    product_page_image_layout = fields.Selection([
+        ('carousel', 'Carousel'),
+        ('grid', 'Grid'),
+        ], default='carousel', required=True,
+    )
+    product_page_image_width = fields.Selection([
+        ('none', 'Hidden'),
+        ('50_pc', '50 %'),
+        ('66_pc', '66 %'), # FIXME: bad name -> 66%
```

Comment:
Why is it a bad name ?

Reply:
Nice catch, it's not anymore 🙂 
I forgot to remove the fixme, the value used to be 75%.